src/Microphysics2M.jl

```diff
@@ -447,6 +447,103 @@ function rain_evaporation(
     return (evap_rate_0, evap_rate_1)
 end
 
+""" 
+    radar_reflectivity(struct, struct, q_liq, q_rai, N_liq, N_rai, ρ_air, ρ_w)
+
+    - `struct` - type for 2-moment rain autoconversion parameterization
+    - `νr` - rain gamma distribution mass coefficient
+    - `μr` -  rain gamma distribution exponential coefficient 
+    - `q_rai` - rain water specific humidity
+    - `N_liq` - cloud droplet number density 
+    - `N_rai` - rain droplet number density 
+    - `ρ_air` - air density
+    - `ρ_w` - water density
+
+Returns logarithmic radar reflectivity from the assumed cloud and rain particle 
+size distribuions normalized by the reflectivty of 1 millimiter drop in a volume of
+one meter cube
+"""
+function radar_reflectivity(
+    acnv::CMP.AcnvSB2006{FT},
+    νr::FT,
+    μr::FT,
+    q_liq::FT,
+    q_rai::FT,
+    N_liq::FT,
+    N_rai::FT,
+    ρ_air::FT,
+    ρ_w::FT,
+) where {FT}
+
+    (; νc) = acnv
+
+    xc = (N_liq == 0) ? 0 : ((q_liq * ρ_air) / N_liq)
+    xr = (N_rai == 0) ? 0 : ((q_rai * ρ_air) / N_rai)
+    C = FT((4 / 3) * π * ρ_w)
+    Z₀ = FT(1e-18)
+
+    Bc = (xc == 0) ? 0 : (SF.gamma(νc + 1) * xc / SF.gamma(νc + 2))^(-1)
+    Br =
+        (xr == 0) ? 0 :
+        (SF.gamma((νr + 1) / μr) * xr / SF.gamma((νr + 2) / μr))^(-μr)
+    Ac = (N_liq * Bc^(νc + 1)) / SF.gamma(νc + 1)
+    Ar = (μr * N_rai * Br^((νr + 1) / μr)) / SF.gamma((νr + 1) / μr)
+
+    Zc = (Bc == 0) ? 0 : 10 * (log10(24 * Ac / (Bc^5 * C^2 * Z₀)) + log10(1e-9))
+    Zr =
+        (Br == 0) ? 0 :
+        10 * (log10(2160 * Ar / (Br^7 * C^2 * Z₀)) + log10(1e-9))
+
+    return ((N_liq + N_rai) == 0) ? -200 :
```

Comment:
I'm not sure about the number weighted approach here. It's fine to merge it in, but lets test the other idea